polyfill/test/validStrings.mjs

```diff
@@ -462,7 +443,7 @@ const comparisonItems = {
   ],
   MonthDay: ['month', 'day', 'calendar'],
   Time: [...timeItems],
-  TimeZone: ['offsetMinutes', 'tzName'],
+  TimeZone: ['offset', 'tzAnnotation'],
```

Comment:
This was missing a `z` I think. Fixed in the latest commit. Also added the `z` to Instant and ZDT.